test/unit/spec/IDPDiscovery_spec.js

```diff
@@ -634,7 +640,7 @@ Expect.describe('IDPDiscovery', function() {
     });
     itp('toggles "focused-input" css class on focus in and focus out', function() {
       return setup().then(function(test) {
-        test.form.usernameField().focus();
+        test.form.usernameField().focusin();
```

Comment:
Can you help me understand why `focus` doesn't work anymore?

Reply:
Not entirely sure. May be related to this: https://github.com/jquery/jquery/issues/4382